cmd/loop/liquidity.go

```diff
@@ -173,6 +174,131 @@ func setRule(ctx *cli.Context) error {
 	return err
 }
 
+var setParamsCommand = cli.Command{
+	Name:        "setparams",
+	Usage:       "update the parameters set for the liquidity manager",
+	Description: "Updates the parameters set for the liquidity manager.",
+	Flags: []cli.Flag{
+		cli.Float64Flag{
+			Name: "maxswapfee",
+			Usage: "the maximum ratio of swap volume we are " +
+				"willing to pay in server fees (eg, 0.01 = " +
+				"1%).",
```

Comment:
This explanation, doesn't this hint that we need to accept (float) percentages? It is easy to make a mistake with all these zeroes.